scripts/parse_performance_metrics.py

```
@@ -0,0 +1,200 @@
+`flutter test --machine`, but plain-text logs containing a line that begins
+with `PERFORMANCE_METRICS:` are also supported.
+"""
+
+from __future__ import annotations
+
+import argparse
+import json
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Dict, Iterable, Tuple
```

Ruff (0.14.0)

Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)